packages/hdwallet-core/src/utils.ts

```diff
@@ -253,3 +253,8 @@ export function compatibleBufferConcat(list: Uint8Array[]): Buffer {
 export function isIndexable(x: unknown): x is Record<string | number | symbol, unknown> {
   return x !== null && ["object", "function"].includes(typeof x);
 }
+
+/* Replacement for _.isObject() */
+export function isObject(x: any) {
+  return typeof x === "object" && x !== null;
+}
```

Comment:
1:1 replacement assuming there wasn't an issue with the function typeof check?
```suggestion
/* Replacement for lodash.isObject() */
export function isObject(x: unknown) {
  return x !== null && ["object", "function"].includes(typeof x);
}
```